### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by jerrycode06**
Modified site with version 2 theme of [Hugo Grayscale](https://github.com/runningstream/hugograyscale).

### CODE REVIEW

**Comment by jerrycode06** [comment]
These are the `public` folder files, Should I also include the whole project files in this PR without the `themes` folder? 
@pnijhara

**Comment by pnijhara** [comment]
Exactly!
The whole point is to push the whole project along with the `public` folder. And target the `public` folder for GitHub pages to build. 
@prateekj117 can you confirm this?

**Comment by RJ722** [comment]
You can use a GitHub action to build the site for each commit (on master) and push the built site to the gh-pages branch — which can then be deployed using GitHub Actions.

Something like https://github.com/rj722/red

**Comment by pnijhara** [comment]
@RJ722's point is also valid. We can push the Hugo project in the `master` branch and the `public` folder in the `gh-pages` branch.

**Comment by jerrycode06** [comment]
> 
> 
> You can use a GitHub action to build the site for each commit (on master) and push the built site to the gh-pages branch — which can then be deployed using GitHub Actions.
> 
> Something like https://github.com/rj722/red

This one also looks good.

File: content/sections/section2.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: "Blog"
+weight: 3
+improvecontrast: true
+---
+
+## AMU-OSS Blog
+
+You can read our blogs here.
+
+{{< big-button text="BLOG" href="https://testamuossblog.tk/read" >}}
```
Comment (pnijhara) [inline content/sections/section2.md:11]: This blog link won't work.

File: config.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+baseURL = "http://example.com/"
```
Comment (pnijhara) [inline config.toml:1]: baseURL should be http://amu-oss.github.io/

File: content/sections/section2.md (hunk 1 above)
Comment (jerrycode06) [reply content/sections/section2.md:11]: what should be the link here?

Comment (pnijhara) [reply content/sections/section2.md:11]: @RJ722 should we add the blog link? I don't think the blog website you created is still working.

**Review by pnijhara — APPROVED** [review]

**Pull request merged**